app/assets/images/payment_icons/airteltigo.svg

```diff
@@ -0,0 +1 @@
.2c-.4.3-1 .4-1.5.4s-1-.4-1.2-.9c-.2-.4-.3-.8-.3-1.2 0-.4.1-.7.3-1 .2-.4.5-.7 1-.7.3-.1.7 0 1 .1.3.1.5.3.6.6.1.4.2.8.2 1.3zm-1-.6c0-.2 0-.4-.2-.6-.2-.1-.4-.1-.6 0-.2.2-.3.4-.3.7.4-.1.7-.1 1.1-.1zm-4-2.3l1-.2v1h.6v.7h-.5c-.1 0-.1 0-.1.1v1.9c0 .2.2.4.4.4h.2v.6c-.3.1-.7 0-1-.1-.2-.1-.4-.2-.5-.4-.1-.2-.1-.4-.1-.6V8zm7.1 4v.6h-.1c-.3.1-.6 0-.9-.1-.3-.1-.5-.4-.5-.7 0-.2-.1-.3-.1-.5V8l1-.2v3.9c0 .1.1.3.3.3h.3zm-7.3-3.1l-.3.6h-.2c-.2 0-.5 0-.7.1-.1 0-.2.1-.1.2v2.7h-1V9.7c0-.1 0-.2.1-.2.4-.4 1-.6 1.5-.6.2 0 .3 0 .5.1.1-.1.1-.1.2-.1zm-3 3.7H12V9.2c0-.1 0-.1.1-.2l.6-.1h.2v3.7zm.1-4.7c0 .3-.2.6-.5.6s-.6-.2-.6-.6c0-.3.3-.6.6-.6.2.1.5.3.5.6z" fill="#ff000d"/><path d="M32.2 10.9c0-.4.1-.7.3-1 .2-.4.7-.7 1.1-.8.3-.1.7 0 1 .1.5.2.8.5 1 1 .2.6.2 1.2-.1 1.8-.3.6-.8.9-1.4.9-.9.1-1.6-.5-1.8-1.4-.1-.3-.1-.5-.1-.6zm.9-.1c0 .2 0 .4.1.6.1.4.6.6 1 .5.2-.1.4-.2.5-.5.1-.2.1-.5.1-.7 0-.2-.1-.4-.1-.5-.1-.3-.4-.5-.8-.4-.3 0-.6.2-.6.5-.2.2-.2.3-.2.5zm-1.4-.7h-.1c-.4-.2-.8-.3-1.2-.3-.6.1-1 .5-1 1.1 0 .2 0 .4.1.6.1.3.4.6.8.6h.5c.1 0 .2-.1.3-.2v-1h.6c.2 0 .3.2.4.4v.7c0 .4-.3.7-.6.8-.6.2-1.3.1-1.8-.2-.5-.3-.8-.8-.8-1.3-.1-.5 0-1 .3-1.4.3-.4.7-.6 1.1-.7h.8c.2 0 .4.1.6.2.2.1.3.3.3.6-.3-.1-.3 0-.3.1zm-7-1.8v.8h1.2v.4c0 .2-.2.3-.4.3h-.8c-.1 0-.1 0-.1.1v1.4c0 .3.2.5.4.5s.5 0 .7-.1h.1c.1.3 0 .7-.3.8h-.2c-.3 0-.6 0-.8-.1-.4-.1-.7-.4-.7-.8 0-.2-.1-.3-.1-.5V8.8c.1-.2.2-.3.3-.4.3-.1.5-.2.7-.1zm-6.4 5.9c0 .1-.1.3-.1.4l-.4 1.2c0 .1-.1.1-.1.1h-.4c-.1 0-.1 0-.1-.1l-.5-1.5v-.1l-.2 1.8H16c0-.3.1-.6.1-.9s.1-.6.1-.9.1-.6.1-.9h.5c.1 0 .1 0 .1.1l.3.9.3.8c0 .1 0 .2.1.2l.2-.7c.1-.2.1-.5.2-.7l.2-.7h.5l.4 2.6h-.6l-.2-1.6zm11.1-.8c.5.8 1.4 1.3 2.4 1.3s1.9-.5 2.4-1.3v.6c-.1.3-.2.6-.5.9-1.1 1-2.9 1-3.9-.1l-.1-.1c-.2-.2-.3-.6-.3-.9v-.4zm-2.8-4.2h.4c.2 0 .4.2.4.5v2c0 .1.1.3.3.3h.4v.5c0 .1-.1.2-.2.2-.5.1-.9 0-1.2-.5-.1-.2-.1-.3-.1-.5V9.2zm-.2 7.6v-.4h.2c.2 0 .3-.1.4-.3 0-.1 0-.1.1-.2v-.3c-.2-.5-.4-1-.6-1.6 0-.1 0-.1-.1-.2h.5l.1.1.3 1 .1.1v-.1c.1-.3.2-.7.3-1 0-.1 0-.1.2-.1h.4v.1c-.3.7-.5 1.4-.8 2.1 0 .1-.1.3-.2.4-.2.2-.5.3-.7.3-.1.1-.1.1-.2.1zM19.6 15c0-.2 0-.3.1-.5.1-.3.3-.5.6-.5.2-.1.4-.1.6 0 .3.1.6.4.6.8 0 .2 0 .4-.1.6-.1.4-.5.7-1 .6-.3 0-.6-.2-.8-.5V15zm1.4 0c0-.2 0-.3-.1-.5-.1-.1-.2-.2-.4-.2s-.3.1-.3.3v.2c0 .2 0 .4.1.6.1.1.2.2.4.2s.3-.1.3-.3c-.1-.1-.1-.2 0-.3zm2.8.9h-.5v-1.4c0-.1-.1-.2-.2-.2H23c-.1 0-.2.1-.3.2v1.3h-.5v-1.9h.5v.2l.1-.1c.2-.2.4-.2.7-.1.2 0 .3.2.4.4v1.4c-.1.1-.1.2-.1.2zm2.3-.8h-1.2c0 .2.1.4.2.5.2.1.3.1.5 0l.3-.1.2.3-.1.1c-.3.1-.6.2-1 .1-.3-.1-.5-.3-.6-.5-.1-.3-.1-.6 0-.8.2-.4.6-.7 1-.6.4 0 .6.3.6.7.1 0 .1.1.1.3zm-.5-.4c0-.1 0-.3-.2-.4-.1-.1-.3-.1-.4 0-.1.1-.2.2-.2.3l.8.1zm9.9-6.1c-.4-.5-1-.7-1.6-.7-.6 0-1.2.3-1.6.7v-.4c0-.2.1-.4.3-.6.3-.2.7-.4 1.1-.4.5-.1 1.1.1 1.5.5.2.2.3.4.3.7.1 0 .1.1 0 .2zm-8-.3c0 .3-.2.5-.5.5s-.5-.2-.5-.5.2-.5.4-.6c.3 0 .5.2.6.4v.2z" fill="#143e7b"/><path d="M34 .8c1.8 0 3.2 1.5 3.2 3.2v16c0 1.8-1.5 3.2-3.2 3.2H4C2.2 23.2.8 21.7.8 20V4C.8 2.2 2.3.8 4 .8h30m0-.8H4C1.8 0 0 1.8 0 4v16c0 2.2 1.8 4 4 4h30c2.2 0 4-1.8 4-4V4c0-2.2-1.8-4-4-4z" fill="#212121" opacity=".07"/></svg>
```

Comment:
The border radius doesn't look the same compared to other icons.

Also, can we remove `id="Layer_1"`? It's not used elsewhere.

[Video demonstration](https://screenshot.click/13-10-1dhy3-9iyyv.mp4)